ci-operator/step-registry/cucushift/hypershift-extended/install-private/cucushift-hypershift-extended-install-private-chain.yaml

```diff
@@ -3,6 +3,9 @@ chain:
   steps:
   - ref: cucushift-hypershift-extended-install-private-config
   - ref: cucushift-hypershift-extended-install-private
+  env:
+  - name: HYPERSHIFT_AWS_REGION
+    default: ""
```

Comment:
We can remove this, step: cucushift-hypershift-extended-install-private-config, `HYPERSHIFT_AWS_REGION` has already been defined.

Reply:
Done.